post_update.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+echo "Initialising db"
+service postgresql initdb
+
+echo "Starting postgresql"
+service postgresql start
+
+USER="pgsql"
+PG_CLIENT="/usr/local/bin/psql"
+PG_HOST="/tmp"
+DEST_DIR="/usr/local/ds-system"
+PASS=`cat /root/dbpassword`
+DUMP_FILE_PATH="/tmp/update_dump"
+
+export LC_ALL=C
```

Comment:
Doesnt en_US.UTF-8 work?

Reply:
It does, copied it from `post_install` script, thought it might be required for some case I'm missing. Should we keep or should i update ?

Reply:
I think using UTF-8 is more future proof